server/src/test/java/com/google/android/attestation/ParsedAttestationRecordTest.java

```diff
@@ -91,7 +91,7 @@ private static X509Certificate getAttestationRecord(String certStr) throws Certi
   public void testParseAttestationRecord() throws CertificateException, IOException {
     X509Certificate x509Certificate = getAttestationRecord(CERT);
     ParsedAttestationRecord attestationRecord =
-        ParsedAttestationRecord.createParsedAttestationRecord(x509Certificate);
+        ParsedAttestationRecord.createParsedAttestationRecord(new X509Certificate[] {x509Certificate});
```

Comment:
Could you modify this test to pass in two certificates, where the current is expected and another is the leaf? I'll send you a generated certificate to use - since the chain validity is not checked in the parsing code it shouldn't matter that we don't have the original subject key.

Reply:
yeah, we can modify getAttestationRecord to takes multiple certs.
getAttestationRecord(String[] certStr) ? or X509Certification[] ?